### PR TITLE
[No Functional Changes] Proper comment indentation

### PR DESCRIPTION
**Pull request by jacob-hughes**
I recently set my vim indentation to 80 character width and it looks better so I'd like to merge these changes into master please.

### CODE REVIEW

File: src/interp.rs [hunk 1]
```diff
@@ -37,9 +36,9 @@
 
 use rustc::ty::Ty;
 
-// Stores a value represented as a vector of bytes, along with its type annotation.
-// It's important to keep track of a value's type while it is being evaluated,
-// so that we know how to interpret the bytes.
+// Stores a value represented as a vector of bytes, along with its type
+// annotation.  It's important to keep track of a value's type while it is being
```
Comment (ltratt) [inline src/interp.rs]: Double space after "."

**Comment by ltratt** [comment]
Just the one minor comment.

File: src/interp.rs (hunk 1 above)
Comment (jacob-hughes) [reply src/interp.rs]: Ah, I think this was a rogue vim setting. `set nojoinspaces` ;)

**Comment by jacob-hughes** [comment]
Fixed

**Comment by jacob-hughes** [comment]
Am I good to squash this so that it can be merged?

**Comment by ltratt** [comment]
Yep, please squash.

**Comment by jacob-hughes** [comment]
Squashed!

**Pull request merged**